packages/create-catalyst/src/utils/localization.ts

```diff
@@ -0,0 +1,51 @@
+import chalk from 'chalk';
+
+import { Https } from './https';
+
+interface Locale {
+  fallback: string | null;
+  id: string;
+  is_supported: boolean;
+  name: string;
+}
+
+interface AvailableLocalesResponse {
+  data: Locale[];
+}
+
+const isAvailableLocalesResponse = (response: unknown): response is AvailableLocalesResponse =>
+  typeof response === 'object' &&
+  response !== null &&
+  'data' in response &&
+  Array.isArray(response.data) &&
+  response.data.every(
+    (item) =>
+      typeof item === 'object' &&
+      item !== null &&
+      'id' in item &&
+      'name' in item &&
+      'fallback' in item &&
+      'is_supported' in item,
+  );
```

Comment:
Can we use `zod` to validate/define these types?

Reply:
Yeah, I will update it, to use `zod`